### ISSUE TITLE
Should PUT-to-create c/r in existing container be allowed with only accessTo Write c/?

### ISSUE THREAD

**Issue by michielbdejong** [solid/specification#246]
### The question
Suppose:
* `c/` exists
*  `c/` has its own ACL doc, which looks as follows (leaving out prefix etc):
```turtle
<#Bob>
  a acl:Authorization;
  acl:agent: <https://bob.com/#me> ;
  acl:accessTo <c/>;
  acl:mode acl:Write.
```
* `c/r` does not exist
* Bob (authenticated as https://bob.com/#me) does a PUT to c/r

Note that the Bob's Authorization in the ACL doc of `c/` does not contain `acl:default`, so it does not apply to descendants of `c/`, only to `c/` itself.

Should this request succeed and create `c/r`?

### The facts
* ESS says 'yes'.
* NSS, PSS, and CSS say 'no'.
* For comparison, and in support for 'yes', POST to c/ also only requires `accessTo Append`, no default permissions required there either.
* But then again, in support of 'no', with a POST the client doesn't get to choose the URL, and with a PUT they do.

Please vote! :)

**Comment by michielbdejong** [comment]
@timbl @justinwb @dmitrizagidulin @kjetilk @csarven @RubenVerborgh you are in the [editorial team](https://github.com/solid/process/blob/master/editors.md#editorial-team), your voice is obviously very welcome in this discussion.

**Comment by michielbdejong** [comment]
> * ESS says 'yes'.

To reproduce this, have a look at https://github.com/solid/web-access-control-tests/tree/reproduce-246-ess

> * NSS, PSS, and CSS say 'no'.

To reproduce this, unskip https://github.com/solid/web-access-control-tests/blob/main/test/surface/create.test.ts#L207 and you'll see the CRUD tests still pass against each of these three servers.

**Comment by csarven** [comment]
>Should PUT-to-create c/r in existing container be allowed with only accessTo Write c/?

As only acl:Write is granted to C/ and no access can be determined for C/R, access is denied.

**Comment by michielbdejong** [comment]
@acoburn was this fixed in ESS v1.1?

**Comment by csarven** [comment]
Closing this issue as consensus is deemed to be captured in WAC Editor's Draft: https://solid.github.io/web-access-control-spec/ . See #effective-acl-resource #reading-writing-resources #authorization-matching
